src/App.svelte

```diff
@@ -375,6 +405,10 @@
 	html {
     font-size: 8px;
 	}
+
+	body{
```

Comment:
any reason why you added this right here?

Reply:
I added that cause i noticed that the page had default padding which looked weird, so i just removed it.